src/library_html5.js

```diff
@@ -1557,7 +1557,7 @@ var LibraryJSEvents = {
   },
 
   emscripten_set_pointerlockchange_callback: function(target, userData, useCapture, callbackfunc) {
-    if (!document.body.requestPointerLock && !document.body.mozRequestPointerLock && !document.body.webkitRequestPointerLock && !document.body.msRequestPointerLock) {
+    if (!document.body || (!document.body.requestPointerLock && !document.body.mozRequestPointerLock && !document.body.webkitRequestPointerLock && !document.body.msRequestPointerLock)) {
```

Comment:
we could instead define `document.body` in the proxied JS, that seems better as it may fix other things too. And we can add properties to it later as needed. Relevant file is proxyWorker.js